### PR TITLE
[DEV-174669] Implement ClientID in Android SDK

### PR DESCRIPTION
**Pull request by andresguedes**
This PR add cid and sessionCid behavior in the Android SDK.

### CODE REVIEW

File: trustly-android-sdk/publish-remote.gradle [hunk 1]
```diff
@@ -54,9 +54,9 @@ afterEvaluate {
                     }
                     developers {
                         developer {
-                            id = 'andre.guedes@trustly'
-                            name = 'Andre Guedes'
-                            email = 'andre.guedes@trustly.com'
+                            id = 'ops-root@paywithmybank.com'
+                            name = 'TrustlyInc'
+                            email = 'ops-root@paywithmybank.com'
```
Comment (lukevance) [inline trustly-android-sdk/publish-remote.gradle:59]: Shouldn't this be the devX alias? Or is the ops team the right contanct? (amer.developer.experience@trustly.com)

File: trustly-android-sdk/build.gradle [hunk 2]
```diff
@@ -52,10 +61,12 @@ task javadoc(type: Javadoc) {
     exclude '**/TrustlyPanel.java'
 }
 
+apply from: 'publish-root.gradle'
+
 ext {
     PUBLISH_GROUP_ID = 'io.github.TrustlyInc'
     PUBLISH_VERSION = '2.2.1'
```
Comment (lukevance) [inline trustly-android-sdk/build.gradle:68]: Is this version still 2.2.1? I thought this repo was only starting in 3.0.0?

File: .gitignore [hunk 3]
```diff
@@ -83,3 +83,6 @@ lint/generated/
 lint/outputs/
 lint/tmp/
 # lint/reports/
+
+# Mac DS files
+.DS_Store
```
Comment (lukevance) [inline .gitignore:88]: This .DS_Store file appears to still be showing up in the repo, not a big deal, but can we get rid of it?

**Review by lukevance — COMMENTED** [review]
Just a couple small questions, otherwise looks good!

File: trustly-android-sdk/publish-remote.gradle (hunk 1 above)
Comment (andresguedes) [reply trustly-android-sdk/publish-remote.gradle:59]: This is the DevOps team contact, @pgonring added this info when we publish the first version to the Maven Central

File: trustly-android-sdk/build.gradle (hunk 2 above)
Comment (andresguedes) [reply trustly-android-sdk/build.gradle:68]: In this PR yet! There's another PR just to change the version code

File: .gitignore (hunk 3 above)
Comment (andresguedes) [reply .gitignore:88]: Yes, I'll change it!

**Review by lukevance — APPROVED** [review]
LGTM! 👍

**Review by pedroabdenor — APPROVED** [review]
👍

**Comment by rhaissatrustly** [comment]
LGTM 👍

**Pull request merged**